BLSQ/iaso
# IA-2291: Need a read only permission on top of the existing one for accessing the pages

Explain what problem this PR is resolving
- Don't allow everyone to update or delete a page
Related JIRA tickets : [IA-2291](https://bluesquare.atlassian.net/browse/IA-2291)

## Self proofreading checklist


## Changes
- Added a menupermissions 'iaso_page_write'
- Check if the current user has permissions to delete or to edit page

## How to test
- Create a new user(not an admin one) add all permissions except **edit page** permission
- Login with the user
- Check if the delete and edit icon on pages list is there or not. They should not appear

## Print screen / video
![Peek 2023-07-10 17-18](https://github.com/BLSQ/iaso/assets/19631540/b401b518-55a4-4ec6-bfaa-e849244ef985)



[IA-2291]: https://bluesquare.atlassian.net/browse/IA-2291?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
